deploy/ray-operator/api/v1alpha1/raycluster_types.go

```diff
@@ -86,21 +86,30 @@ type Extension struct {
 	// Pod docker image
 	Image string `json:"image,omitempty"`
 
+	//create operation should have no idList (generate by controller)
+	IdList []string `json:"idList"`
+
 	// Logical groupName for worker in same group, it's used for heterogeneous feature to distinguish different groups.
 	// GroupName is the unique identifier for the pods with the same configuration in one Ray Cluster
 	GroupName string `json:"groupName,omitempty"`
 
 	// Command to start ray
 	Command string `json:"command,omitempty"`
 
+	// Args to start ray
```

Comment:
Please explain why `args` is needed along with `command`. Provide an example in comment will be nice.

Reply:
Remove the args, it's not used